modules/managing-memcached-operator-using-olm.adoc

```diff
@@ -46,27 +50,43 @@ for more information on manually defining a manifest file.
 
 . *Deploy the Operator.*
 
-.. Deploy an Operator by applying the Operator’s manifest to the desired namespace
-in the cluster:
+.. Create an OperatorGroup that specifies the namespaces that the Operator will
+be targeting. Create the following OperatorGroup in the namespace where you will
+create the CSV. In this example, the `default` namespace is used:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: memcached-operator-group
+  namespace: default
+  spec:
+    targetNamespaces:
+    - default
+----
+
+.. Apply the Operator's CSV manifest to the desired namespace in the cluster:
```

Comment:
s/desired/specified

IBM Style says:
> :no_entry_sign: desired adj
Do not use. Use “required” or “the <item> that you want.”